internal/pkg/keybindings/listhandlers.go

```diff
@@ -432,13 +435,12 @@ func (h ListUpdateHandler) getEditorConfig() (config.EditorConfig, error) {
 		return userConfig.Editor, nil
 	}
 	// generate default config
-	translateFilePathForWSL := wsl.IsWSL() // If on WSL then translate path so that it is valid when loaded by VS code in Windows
 	return config.EditorConfig{
 		Command: config.CommandConfig{
 			Executable: "code",
 			Arguments:  []string{"--wait"},
 		},
-		TranslateFilePathForWSL: translateFilePathForWSL,
+		TranslateFilePathForWSL: false, // previously used wsl.IsWSL to determine whether to translate path, but VSCode  now performs translation from WSL (so we get a bad path if we have translated it)
```

Comment:
nit: If we no longer need this can we remove the property from the `EditorConfig` struct?

Ah wait I've misunderstood this is only when using VSCode... gotcha

Reply:
Yup, still needed in config :-)